app/Models/User.php

```diff
@@ -9,6 +9,7 @@
 use Illuminate\Contracts\Auth\MustVerifyEmail;
 use Illuminate\Database\Eloquent\Factories\HasFactory;
 use Illuminate\Database\Eloquent\SoftDeletes;
+use Illuminate\Database\Eloquent\Model;
```

Comment:
Do we need to include Model here ?

Reply:
Apologies for the mistake. You are right, we don't need to include the model here. I shall make the necessary changes.